impact.yuml

```diff
@@ -51,3 +54,20 @@
 # // [note: Cisco UCS;ZenPack{bg:darkslateblue}]-[UCS Service Profile{bg:darkslateblue}]
 [UCS Service Profile{bg:darkslateblue}]1-*>[Linux;Device{bg:deepskyblue}]
 
+<<<<<<< Updated upstream
+=======
+# Cinder
+[CinderAgent{bg:orange}]*-*>[Volume{bg:orange}]
```

Comment:
There doesn't appear to be a component in your model called a cinder agent.  You may need to enhance the model, or change the way the impact works.

I would also be concerned about the cardinality on this- if there is an agent of some sort on every host, you don't want to get into a situation where if any host goes down, every volume is impacted.